Update tiflash-results-materialization.md (#15632)
pingcap/docs: tiflash/tiflash-results-materialization.md

('2023-1-3 12:2:3', 'cus002', 2200.86),
('2023-1-4 12:2:3', 'cus003', 2020.86),
('2023-1-5 12:2:3', 'cus003', 1200.86),
('2023-1-6 12:2:3', 'cus002', 20.86),
('2023-1-7 12:2:3', 'cus004', 120.56),
('2023-1-8 12:2:3', 'cus005', 320.16);

> -- repeate this sql 13 times, make that the table has 65536 rows

Reviewer: ```suggestion
-- Repeat executing the following SQL statement 13 times, which inserts a total of 65,536 rows to the table.
```

> -- Repeat executing the following SQL statement 13 times, which inserts a total of 65,536 rows to the table.

Reviewer 2: ```suggestion
-- Execute the following SQL statement 13 times to insert a cumulative total of 65,536 rows into the table.
```

> INSERT INTO detail_data select * from detail_data;

Reviewer 2: ```suggestion
INSERT INTO detail_data SELECT * FROM detail_data;
```
